site/src/scaffold/scripts/tree-example.ts

```diff
@@ -0,0 +1,158 @@
+import {
```

Comment:
Remove? This file seems ... accidentally committed?

Reply:
Yes it was by accident, removed.